src/pages/documentation.html

```diff
@@ -139,8 +139,11 @@
 					</div>
 				</div>
 			</div>
-			<div class="section mod_comparison">
+			<div class="section mod_comparison color_lightblue">
```

Comment:
border-top should be removed then

Reply:
But why can this be used in Projects over the footer but not here over the footer?